src/main/scala/mesosphere/marathon/core/matcher/base/util/OfferOperationFactory.scala

```diff
@@ -93,7 +93,7 @@ class OfferOperationFactory(
       Mesos.Resource.newBuilder()
         .setName("disk")
         .setType(Mesos.Value.Type.SCALAR)
-        .setScalar(Mesos.Value.Scalar.newBuilder().setValue(vol.persistentVolume.persistent.size.toDouble).build())
+        .setScalar(Mesos.Value.Scalar.newBuilder().setValue(vol.persistentVolume.persistent.size.get.toDouble).build())
```

Comment:
in this case validation would have failed if size wasn't specified. but we could probably also fold or reduce here instead of using get